src/main/java/io/cryostat/net/SslConfiguration.java

```diff
@@ -111,7 +115,7 @@ public class SslConfiguration {
         }
 
         strategy = new NoSslStrategy();
-        logger.info("Selected NoSSL strategy");
+        logger.info("++-Selecteeeeeeeeeed NoSSL strategy");
```

Comment:
Probably don't want to include this change

Reply:
This was to be able to keep track with ctrl f